.gitignore

```diff
@@ -6,3 +6,4 @@ build
 packages/*/npm-debug.log.*
 es
 dist
+.DS_Store
```

Comment:
This is platform specific. Do we need it? I added a global ignore for this. http://technicalpickles.com/posts/configure-git-to-globally-ignore-some-files/